tools/make_class_power.py

```diff
@@ -0,0 +1,206 @@
+"""This module creates a matter power spectrum using Classylss,
+a python interface to the CLASS Boltzmann code.
+
+See:
+http://classylss.readthedocs.io/en/stable/
+http://class-code.net/
+It parses an MP-GenIC parameter file and generates a matter power spectrum
+and transfer function at the initial redshift. It generates a second transfer
+function at a slightly lower redshift to allow computing the growth function.
+Files are saved where MP-GenIC expects to read them. Existing files will not be over-written.
+
+Script should be compatible with python 2.7 and 3.
+
+Cite CLASS paper:
+ D. Blas, J. Lesgourgues, T. Tram, arXiv:1104.2933 [astro-ph.CO], JCAP 1107 (2011) 034
+Call with:
+    python make_class_power.py <MP-GenIC parameter file> <external power spectrum file>
+    where the second external power spectrum file is optional and is a primordial power spectrum for CLASS."""
+
+from __future__ import print_function
+import sys
+import math
+import os.path
+import numpy as np
+import classylss
+import classylss.binding as CLASS
+import configobj
+import validate
```

Comment:
What are these two packages?

Are they in general installed? On my vanilla conda (with ipython notebook / jupyter) they do not seem to be installed; I am a bit concerned on this dependency.

Reply:
BTW; why didn't we just use a yaml file for parameters (which embeds nicely in C and Python).